templates/themes/app/parts/content_fields.html.twig

```diff
@@ -42,7 +42,22 @@
                         )
                     ) }}
                 {% else %}
-                    <a href="{{ path("ibexa.url.alias", { "contentId": field.value.reference }) ~ suffix }}"{% if css_class is not empty %} class="{{ css_class }}"{% endif %}{% if field.value.isTargetLinkInNewTab %} target="_blank"{% endif %}>{{ label }}</a>
+                    {{ render(
+                        controller(
+                            "ng_content::viewAction",
+                            {
+                                'contentId': field.value.reference,
+                                'viewType': 'link',
+                                'layout': false,
+                                'params': {
+                                    'label': label,
+                                    'suffix': suffix,
+                                    'css_class': css_class|default(''),
+                                    'new_tab': field.value.isTargetLinkInNewTab
```

Comment:
Please be more explicit here, `open_in_new_tab` instead of just `new_tab`.

Reply:
fixed in 8c01bcc50ab7d22ef0f662c0e98277f6ed9d06d9